main.go

```diff
@@ -88,7 +75,105 @@ func main() {
 			log.Error(err)
 		}
 	})
+	if *allClouds {
+		http.HandleFunc("/probe", probeHandler)
+		http.Handle("/metrics", promhttp.Handler())
```

Comment:
Please add a log entry that specifies that is running in probe mode and which clouds were loaded.

Reply:
Added log message
Regarding displaying list of clouds I don't think it worth doing it. Trade off between adding meaningful information vs adding more code looks not beneficial.
If you agree I would rather not doing it